### PR TITLE
Automatic patching

### PR DESCRIPTION
**Pull request by Jana-Marie**
Hey, 
Thanks for the work, very nice themes! I've written a small python program that can patch the files without any further interaction. Makes it much easier to try all themes :3

### CODE REVIEW

File: patch.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import sys
+
+patch=open(sys.argv[1], "r").readlines()
+orig=open(sys.argv[2],"r")
+origlines = orig.readlines()
+
+for idx,oline in enumerate(origlines):
+  subst = oline.split('=')
+  for idx1,pline in enumerate(patch):
+    if subst[0] in pline:
```
Comment (pointhi) [inline patch.py]: I would suggest to make this test more exact, to be sure it does not "fuzzy" match the wrong string

Comment (Jana-Marie) [reply patch.py]: like this?

Comment (pointhi) [reply patch.py]: I think that should work

**Comment by pointhi** [comment]
thanks, I will merge this first version for now

**Pull request merged**